src/apiServices/ClientService.ts

```diff
@@ -139,8 +139,8 @@ export class ClientService {
         return await this.authService.getUsers(queryString)
     }
 
-    async getAnalysesList(): Promise<AnalysisList> {
-        return await this.analysesService.getAnalysesList()
+    async getAnalysesList(groupUUID: string): Promise<AnalysisList> {
```

Comment:
Is this parameter optional?
```suggestion
    async getAnalysesList(groupUUID?: string): Promise<AnalysisList> {
```

Reply:
Yes please see latest commit (I also changed naming convention)